sys/parsoid.js

```diff
@@ -344,7 +392,9 @@ class ParsoidService {
                         body: res.body[format].body
                     };
                     resp.headers.etag = mwUtil.makeETag(rp.revision, tid);
-                    return this.saveParsoidResult(hyper, req, format, tid, res)
+                    return this.saveParsoidResultToLatest(hyper, req, tid, res)
+                    .catch({ status: 412 }, () =>
```

Comment:
@gwicke So the 412 was the closes one to what we need to indicate here. It's coming from here: https://github.com/wikimedia/restbase/pull/836/files#diff-2505eb75397a4cf81edd753431f60036R369

This is the case where we're trying to store something old to the `latest_and_recent` table. So it's kinda a "Precondition failed". If you have an idea of how to better communicate that info - that would be great.